### DISCUSSION TITLE
How to add more header selections in ToC in posts

### DISCUSSION THREAD

**Discussion by james18lpc** [cotes2020/jekyll-theme-chirpy#1975] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

I want the ToC to display more subheadings in each post.

### Operations you have already tried

I tried modifying toc.js locally but it didn't work.

**Comment by kungfux** [comment, marked as answer]
Please check the answer in related [discussion](https://github.com/cotes2020/jekyll-theme-chirpy/discussions/1364#discussioncomment-10812644).
